helm/chart-operator/templates/priorityclass.yaml

```diff
@@ -1,4 +1,5 @@
-{{- if or .Values.chartOperator.cni.install .Values.e2e  }}
+{{- $existing := lookup "scheduling.k8s.io/v1" "PriorityClass" .Release.Namespace "giantswarm-critical" -}}
```

Comment:
Just one remark: PriorityClasses are cluster-scoped.
```
~ k api-resources | grep -E 'PriorityClass|NAME'
NAME                               SHORTNAMES                 APIVERSION                                 NAMESPACED   KIND
priorityclasses                    pc                         scheduling.k8s.io/v1                       false        PriorityClass
```

Reply:
yes but I don't know how it works in the `lookup` function

Reply:
I think [you could pass an empty string as namespace](https://helm.sh/docs/chart_template_guide/functions_and_pipelines/#using-the-lookup-function)

Reply:
ok thanks, changed to empty string